src/components/tools/ManipulatorTool.vue

```diff
@@ -38,38 +39,98 @@ export default defineComponent({
         : viewProxy.value.getInteractorStyle3D();
     });
 
-    const manipulator = props.manipulatorClass.newInstance(options.value);
+    const manipulators: any[] = [];
+    options.value.forEach((opt) =>
+      manipulators.push(props.manipulatorClass.newInstance(opt))
+    );
+
```

Comment:
Instead of doing the initialization here, maybe let the `watch(options, ...)` block do it. You can add a 3rd parameter, `{ immediate: true }`, which is effectively the same as an init call.